src/js/popup.js

```diff
@@ -255,6 +258,17 @@ const Logic = {
       }
       return identity;
     });
+    const containerOrderStorage = await browser.storage.local.get([CONTAINER_ORDER_STORAGE_KEY]);
+    const containerOrder =
+      containerOrderStorage && containerOrderStorage[CONTAINER_ORDER_STORAGE_KEY];
```

Comment:
I'd prefer we stored the sort order in the state key rather than have another duplicated array with cookiestoreid in it.

Reply:
What do you mean by "state key"? are you talking about the state received from `queryIdentitiesState`?